src/azure-cli-testsdk/azure/cli/testsdk/utilities.py

```diff
@@ -157,18 +151,48 @@ def process_response(self, response):
         return response
 
 
-class AADGraphUserReplacer:
+class MSGraphClientPasswordReplacer(RecordingProcessor):
+    """Replace 'secretText' property in 'addPassword' API's response."""
+
+    PWD_REPLACEMENT = 'replaced-microsoft-graph-password'
+
+    def __init__(self):
+        self._activated = False
```

Comment:
What's purpose for `_activated`?

Reply:
This way, we can make this replacer's `process_response` only be called for `/addPassword` API, saving some time. I kept the original design of `GraphClientPasswordReplacer`.